scripts/ci-scripts/ci-eslint.sh

```diff
@@ -0,0 +1,4 @@
+#!/bin/bash
+
+npm ci --ignore-scripts
```

Comment:
This will make it impractical to run this script locally. I'd suggest moving this to the invoking workflow instead.

```suggestion
```